### PR TITLE
Add dish service

### PR DESCRIPTION
**Pull request by mazen930**

### CODE REVIEW

File: models/dish.go [hunk 1]
```diff
@@ -4,7 +4,6 @@ import "gorm.io/gorm"
 
 type Dish struct {
 	gorm.Model
-	Id        uint `gorm:"primaryKey;autoIncrement"`
-	Name      string
-	ImagePath *string
+	Id   uint `gorm:"primaryKey;autoIncrement"`
```
Comment (phella) [inline models/dish.go]: ID

File: models/dish.go [hunk 2]
```diff
@@ -4,7 +4,6 @@ import "gorm.io/gorm"
...
+	Name string
```
Comment (phella) [inline models/dish.go:8]: why we removed imagePath?

File: routes/dish/router.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+		http.Error(w, "invalid dish id", http.StatusBadRequest)
+		return
+	}
+	res := r.dishService.GetDish(id)
+
+	_ = httputils.JSON(w, res)
+}
+
+func (r dishRouter) createDish(w http.ResponseWriter, req *http.Request) {
+	name := req.FormValue("name")
```
Comment (phella) [inline routes/dish/router.go:51]: this worked? what type of post request you sent?

File: routes/dish/router.go [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+func (d dishRouter) RegisterRoutes() chi.Router {
+	router := chi.NewRouter()
+	router.Get("/{id}", d.getDish)
+	router.Post("/", d.createDish)
+	return router
+}
+
+func (r dishRouter) getDish(w http.ResponseWriter, req *http.Request) {
```
Comment (phella) [inline routes/dish/router.go:38]: I don't see the client will need to fetch a dish by id anywhere.

**Review by phella — APPROVED** [review]

File: models/dish.go (hunk 2 above)
Comment (mazen930) [reply models/dish.go:8]: Cause it become on meal , as per as last schema

File: routes/dish/router.go (hunk 3 above)
Comment (mazen930) [reply routes/dish/router.go:51]: Yes , it worked

File: routes/dish/router.go (hunk 4 above)
Comment (mazen930) [reply routes/dish/router.go:38]: Not needed for now , but just in case will remove it later if not needed

**Pull request merged**